### DISCUSSION TITLE
Add traffic lights to flows

### DISCUSSION THREAD

**Discussion by schophil** [postmanlabs/postman-flows#61] [Ideas]
Would be cool to have a traffic light component that can be coupled to the "test" output of requests and that shows a green light when all is ok and a red light when a test fails. I miss the visual feedback that something fails.

**Comment by samay-v** [comment]
@schophil Thats an interesting analogy 😄 
We do intend to look into expanding the test result section, but I don't have a solid timeline for you yet.

**Comment by anudeepmedicharla** [comment]
can probably be bucketed under "visualizers" within Flows.
